Frontend/ui-customer/src/Components/DriverInfoCard/View.purs

```diff
@@ -840,10 +840,13 @@ driverDetailsView push state =
           , gravity LEFT
           ] <> FontStyle.body7 TypoGraphy
         , textView (
-          [ text (state.data.vehicleDetails <> case state.data.vehicleVariant of
-                          "TAXI_PLUS" -> " (" <> (getString AC_TAXI) <> ")"
-                          "TAXI" -> " (" <> (getString NON_AC_TAXI) <> ")"
-                          _ -> "")
+          [ text $ if state.data.vehicleDetails == "" then "" else state.data.vehicleDetails <> " · " 
+                    <> case getMerchant FunctionCall of
+                          YATRISATHI -> getVariantRideType state.data.vehicleVariant
+                          _          -> case state.data.vehicleVariant of
+                                          "TAXI_PLUS" -> " (" <> (getString AC_TAXI) <> ")"
```

Comment:
@ashishsingh101 we will be getting TAXI_PLUS and TAXI variant in YATRISATHI alone so this case match can be removed and we can just send `""` and appending`" · "`  is required only in YATRISATHI, for other merchants its only the vehicleDetails which is visible